var/spack/repos/builtin/packages/py-fenics-basix/package.py

```diff
@@ -15,22 +15,25 @@ class PyFenicsBasix(PythonPackage):
     maintainers("chrisrichardson", "mscroggs", "garth-wells", "jhale")
 
     version("main", branch="main")
+    version("0.7.0", sha256="9bee81b396ee452eec8d9735f278cb44cb6994c6bc30aec8ed9bb4b12d83fa7f")
     version("0.6.0", sha256="687ae53153c98facac4080dcdc7081701db1dcea8c5e7ae3feb72aec17f83304")
     version("0.5.1", sha256="69133476ac35f0bd0deccb480676030378c341d7dfb2adaca22cd16b7e1dc1cb")
     version("0.4.2", sha256="a54f5e442b7cbf3dbb6319c682f9161272557bd7f42e2b8b8ccef88bc1b7a22f")
 
     depends_on("fenics-basix@main", type=("build", "run"), when="@main")
+    depends_on("fenics-basix@0.7.0", type=("build", "run"), when="@0.7.0")
     depends_on("fenics-basix@0.6.0", type=("build", "run"), when="@0.6.0")
     depends_on("fenics-basix@0.5.1", type=("build", "run"), when="@0.5.1")
     depends_on("fenics-basix@0.4.2", type=("build", "run"), when="@0.4.2")
 
+    depends_on("cmake@3.19:", type="build")
+
+    depends_on("python@3.7:", when="@:0.6.0", type=("build", "run"))
+    depends_on("python@3.8:", when="@0.7.0:", type=("build", "run"))
     depends_on("py-setuptools@42:", type="build")
-    depends_on("py-setuptools@40:", type="build")
     depends_on("py-numpy@1.21:", type=("build", "run"))
-    depends_on("cmake@3.19:", type="build")
-    depends_on("python@3.7:", type=("build", "run"))
-    depends_on("py-pybind11@2.9.1:", type="build")
-    depends_on("py-pybind11@2.2.4:", type="build")
+    depends_on("py-pybind11@2.9.1:", when="@:0.7", type="build")
```

Comment:
Missing dep on py-scikit-build-core

Reply:
`py-scikit-build-core` is only used for 'convenience' pip install that builds the C++ library and the python interface in one step.

The Spack packages separate the C++ library and the and the Python wrappers, and don't require `py-scikit-build-core`. The Python requirements are listed in `python/` directory. Python requirements in the top level directory aren't relevant for the Spack packages.

Reply:
Ah, can you add a comment like:
```python
# python/pyproject.toml
```
just so I remember where to look for deps when I review next time?

Reply:
Just waiting on this comment